ocp_resources/virtual_machine_snapshot_content.py

```diff
@@ -0,0 +1,10 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class VirtualMachineSnapshotContent(NamespacedResource):
```

Comment:
Please add to_dict method with the required values to create  VirtualMachineSnapshotContent

Reply:
VirtualMachineSnapshotContent is created by the VM snapshot controller, we don't create it ourselves.